docker/docker-compose.yml

```diff
@@ -37,8 +40,8 @@ services:
       - "${PORT_CS_WORDPRESS:-8000}:80"
     restart: always
     extra_hosts:
-      - "api.crowdsignal.com:${CS_SANDBOX_IP:-192.0.123.248}"
-      - "app.crowdsignal.com:${CS_SANDBOX_IP:-192.0.123.248}"
+      - "api.crowdsignal.com:${CS_SANDBOX_IP:-192.0.123.178}"
```

Comment:
These 2 lines need to remain as they were.

Reply:
I've corrected those lines